completed assignment 2

geoffreykithuku opened the pull request.

geoffreykithuku committed:
complete

Jkuffler commented:
Your ruby code has a bit of extra structure causing it to _not_ run as expected. Simply follow the example given in the instructions which do not include opening `def` and `end` statements. To check your work run the ruby file 👍 

SQL queries look solid!

geoffreykithuku committed:
fixed ruby assignment

geoffreykithuku commented:
> Your ruby code has a bit of extra structure causing it to _not_ run as expected. Simply follow the example given in the instructions which do not include opening `def` and `end` statements. To check your work run the ruby file 👍
> 
> SQL queries look solid!



I had used the methods structure and forgot to call them. I have fixed the issue and removed the method definitions.
